cmd/monoctl/get/audit_log.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 Monoskope Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package get
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/finleap-connect/monoctl/cmd/monoctl/flags"
+	"github.com/finleap-connect/monoctl/internal/config"
+	"github.com/finleap-connect/monoctl/internal/usecases"
+	authutil "github.com/finleap-connect/monoctl/internal/util/auth"
+	"github.com/spf13/cobra"
+)
+
+func NewGetAuditLogCmd() *cobra.Command {
+	var (
+		from string
+		to string
+
+		layout = "02.01.2006" // don't change. This corresponds to DD.MM.YYYY
+		now = time.Now()
+		firstOfMonth = time.Date(now.Year(), now.Month(), 1, 0, 0, 0, 0, now.Location())
```

Comment:
The events in the server are in UTC. This looks like local time and I am not sure if that works right?

Reply:
Yeah, I was debating this the whole time but was not sure what to go with. Currently, we are taking the date only so it doesn't matter. But if we decided to support time as well this will be a problem. If you are in e.g. USA -> UTC-Time will be 09:52:04.322644 and USA/EDT-Time will be 05:52:04.322644 but now after rethinking about it you are right the events are stored in UTC so whoever is getting the log

1. have to consider this and write the time in UTC (will note that)
2. or we do the conversation, which makes the output client dependent and the timestamp can't be formatted on the server anymore

what would you suggest? going with UTC makes life easier but considering that the project is open-source I wouldn't like it if I input 5 AM and got 9 AM events

Reply:
It is not absolutely true that it doesn't matter if we have only a date because days start and end differently based on the time zone too.

I mean you could take local time as input and convert it to UTC to query the audit log. The audit log itself should ALWAYS be in UTC. No matter what timezone the requesting client has. Or we simply say the input is expected to be UTC, deal with it.

Reply:
Oh yes, you are right.

Ok, considering that, the audit log itself should ALWAYS be in UTC inputting local time converting it to UTC and not converting it back in the output will lead to confusion IMHO, therefore I went with the second suggestion and noted the UTC enforcement

Reply:
fine with me